octopusdeploy_framework/resource_tenant_project.go

```diff
@@ -52,7 +52,7 @@ func (t *tenantProjectResource) Metadata(ctx context.Context, req resource.Metad
 func (t *tenantProjectResource) Schema(ctx context.Context, req resource.SchemaRequest, resp *resource.SchemaResponse) {
 	resp.Schema = schema.Schema{
```

Comment:
Should this get its own schema class like the rest of the resources?

Reply:
Missed one. Thanks

Reply:
fixed